libs/market-list/src/lib/components/markets-container/market-list-table.tsx

```diff
@@ -30,6 +30,7 @@ export const MarketListTable = forwardRef<
   return (
     <AgGrid
       style={{ width: '100%', height: '100%' }}
+      enableCellTextSelection={true}
```

Comment:
suggestion (non blocking): Can we add this as a default prop to:

libs/ui-toolkit/src/components/ag-grid/ag-grid-lazy-themed.tsx
and
libs/ui-toolkit/src/components/ag-grid/ag-grid-dynamic-themed.tsx

I think you should be able to highlight and copy cells by default on all tables

Reply:
updated!